velox/type/Type.h

```diff
@@ -492,10 +504,10 @@ class Type : public Tree<const std::shared_ptr<const Type>>,
 
   static std::shared_ptr<const Type> create(const folly::dynamic& obj);
 
-  // recursive kind hashing (ignores names)
+  // recursive kind hashing (uses only typeKind)
```

Comment:
nit: perhaps, fix the comment to start with a capital letter and end with a period. Same below.